cpp/models/abm/state.h

```diff
@@ -27,6 +27,18 @@ namespace mio
 namespace abm
 {
 
+/**
+ * virus type in ABM.
+ * can be used as 0-based index
+ */
+enum class VirusType : std::uint32_t
+{
+    NoVirus  = 0,
```

Comment:
I dont understand "NoVirus".

Also how does this go with the "Virus" object? Isnt this doubled? Even if this here is "VirusVariant", wouldnt these change the properties of the "Virus" object?

Reply:
This has been reintroduced as VirusVariant.